RecoTracker/IterativeTracking/python/MixedTripletStep_cff.py

```diff
@@ -172,14 +177,12 @@
 
 # TrackingRegion
 mixedTripletStepTrackingRegionsB = _mixedTripletStepTrackingRegionsCommon.clone(RegionPSet = dict(ptMin=0.6, originHalfLength=10.0))
-for e in [pp_on_XeXe_2017, pp_on_AA_2018]:
-    e.toReplaceWith(mixedTripletStepTrackingRegionsB, 
-                    _globalTrackingRegionWithVertices.clone(RegionPSet=dict(
-                fixedError = 2.5,
-                ptMin = 0.6,
-                originRadius = 1.5
+(pp_on_XeXe_2017 | pp_on_AA_2018).toReplaceWith(mixedTripletStepTrackingRegionsB, 
+                mixedTripletStepTrackingRegionsA.clone(RegionPSet=dict(
+                    fixedError = 2.5,
+                    ptMin = 0.6,
```

Comment:
While this works for you, it doesn't work in general as when neither `pp_on_*` is active, `mixedTripletStepTrackingRegionsA` is `globalTrackingRegionFromBeamSpotFixedZ` which doesn't have `fixedError` parameter. I suggest to follow the existing pattern, i.e. something along (feel free to invent better name)
```
_mixedTripletStepTrackingRegionsCommon_pp_on_HI = _globalTrackingRegionWithVertices.clone(...)
...
(pp_on_XeXe_2017 | pp_on_AA_2018).toReplaceWith(mixedTripletStepTrackingRegionsA, _mixedTripletStepTrackingRegionsCommon_pp_on_HI)
...
(pp_on_XeXe_2017 | pp_on_AA_2018).toReplaceWith(mixedTripletStepTrackingRegionsB, _mixedTripletStepTrackingRegionsCommon_pp_on_HI.clone(
    RegionPSet=dict(
        fixedError = 2.5,
        ptMin = 0.6,
    )
)
```